docs/ios.md

```diff
@@ -12,7 +12,7 @@ cargo install cargo-dinghy
 cargo install cargo-dinghy --force
 ```
 
-### iOS phone 
```

Comment:
My vim setup auto removes trailing white space. I can remove revert this if you'd like.